examples/react-native/package.json

```diff
@@ -7,15 +7,15 @@
     "test": "jest"
   },
   "dependencies": {
-    "react": "16.9.0",
-    "react-native": "0.61.5"
+    "react": "16.11.0",
+    "react-native": "0.62.0"
   },
   "devDependencies": {
     "@babel/core": "*",
     "@babel/preset-env": "*",
     "babel-jest": "*",
     "jest": "*",
-    "metro-react-native-babel-preset": "*",
-    "react-test-renderer": "16.9.0"
+    "metro-react-native-babel-preset": "^0.58.0",
```

Comment:
I've had to. anuelly roll back this in the lock file a few times, and it should be at a specific version anyways, so let's specify it